plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1144,6 +1153,9 @@ public String toString() {
             if (_slot  != null) {
                 netBuilder.append(String.format("<address type='pci' domain='0x0000' bus='0x00' slot='0x%02x' function='0x0'/>\n", _slot));
             }
+            if (queues > 0) {
```

Comment:
Add additional condition to if statement when model is virtio

Reply:
Done